frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -263,6 +263,55 @@ describe('Kubernetes resource CRUD operations', () => {
     });
   });
 
+  describe('Events', () => {
+    const name = `${testName}-pod`;
+
+    it('displays Events all resources', async () => {
+      await browser.get(`${appHost}/k8s/all-namespaces/events`);
+      // await crudView.isLoaded();
+      await browser.wait(until.presenceOf($('.co-sysevent__box')));
+      expect($('.co-sysevent__box').isPresent()).toBe(true);
+    });
+
+    it('creates a new pod resource instance for event testing', async () => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/pods`);
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
+      await yamlView.isLoaded();
+      const content = await yamlView.getEditorContent();
+      const newContent = _.defaultsDeep(
+        {},
+        { metadata: { name, labels: { [testLabel]: testName } } },
+        safeLoad(content),
+      );
+      await yamlView.setEditorContent(safeDump(newContent));
+      leakedResources.add(JSON.stringify({ name, plural: 'pods', testName }));
+      await yamlView.saveButton.click();
+      expect(crudView.errorMessage.isPresent()).toBe(false);
+    });
+
+    it('event view displays created pod event', async () => {
+      await browser.get(`${appHost}/ns/${testName}/events`);
+      //TODO: timing flake here only works when going through debugger
+      await browser.wait(until.presenceOf($('.co-sysevent-stream__status')), 10000);
```

Comment:
@spadgett @benjaminapetersen - Hey guys, this testcase is failing but when I use the debugger it passes.  Any recommendations or thoughts on a better test?